examples/zaza-events/README.md

```diff
@@ -0,0 +1,102 @@
+# zaza-events example
+
+## Framework example
+
+The directory `tests-extended/tests.yaml` provides an example of zaza-events
+using 2 Ubuntu units in a very small bundle.  The bundle is:
+
+```yaml
+series: focal
+applications:
+  ubuntu:
+    charm: cs:ubuntu
+    num_units: 2
+```
+
+The `tests.yaml` for this bundle are:
+
+```yaml
+charm_name: none
+gate_bundles:
+- conncheck-focal
+
+target_deploy_status:
+  ubuntu:
+    workload-status-message-regex: "^$"
+
+tests:
+- zaza.charm_tests.conncheck.tests.TestConncheckIntegration
+
+tests_options:
+  plugins:
+    - zaza.plugins.events.configure
+  zaza-events:
+    log-format: InfluxDB
+    keep-logs: false
+    collection-name: DEFAULT
+    collection-description: ""
+    raise-exceptions: true
+    log-collection-name: "testdb-{bundle}-{date}"
+    modules:
+      - conncheck:
+          source: "git+http://github.com/openstack-charmers/conncheck"
+          raise-exceptions: true
+      - logging:
+          logger-name: DEFAULT
+          log-to-stdout: false
+          log-to-python-logging: true
+          python-logging-level: info
+          logger-name: DEFAULT
+          raise-exceptions: true
+    upload:
+      - type: InfluxDB
+        url: ${TEST_INFLUXDB_URL}
+        # user: ${TEST_INFLUXDB_USER}
+        # password: ${TEST_INFLUXDB_PASSWORD}
+        database: testdb
+        timestamp-resolution: ns
+```
+
+
+This assumes that there is an InfluxDB available at the enviroment varaible
+`${TEST_INFLUXDB_URL}` to upload the records to.
+
+
+The actual test is specified in `zaza/charm_tests/conncheck/tests.py`.
+
+The test can be launched by:
+
+```bash
+$ tox -e func-target-extended -- conncheck-focal
```

Comment:
This seems to be referring to a new tox target but I don't see an example of this target in the readme and I can't find a new tox.ini in the patch